lessons/operator/03/README.md

```diff
@@ -1,370 +1,212 @@
+- [API Keys](#api-keys)
+   - [EXERCISE 3: Create an API Key](#exercise-3-create-an-api-key)
+- [Listing and Viewing Resources](#listing-and-viewing-resources)
+  - [EXERCISE 4: List Sensu Resources](#exercise-4-list-sensu-resources)
+- [Inventory Management](#inventory-management)
+  - [EXERCISE 5: List an Inventory of Nodes](#exercise-5-list-an-inventory-of-nodes)
+- [Monitoring as Code](#monitoring-as-code)
+  - [EXERCISE 6: Exporting and Updating Resource Configurations](#exercise-6-exporting-and-updating-resource-configurations)
 - [Learn more](#learn-more)
 - [Next steps](#next-steps)
 
-## Web App
+## Goals
 
-The Sensu backend process includes a web app that provides a unified view of your observability data (events, entities, and silences) and monitoring configuration (checks, handlers, etc) with user-friendly tools to reduce alert fatigue.
+In this lesson we will introduce the `sensuctl` command-line tool.
+You will learn how to install and configure the tool, then practice performing some essential operations.
 
-### Dashboard
+This lesson is intended for operators of Sensu and assumes you have [set up a local workshop environment][setup_workshop].
 
-The Sensu web app homepage is a dashboard that provides a high level overview of the overall health of the systems under management by Sensu.
-The dashboard includes a summary of active incidents, a breakdown of incidents by severity, information about entities under management, and a high level overview of the namespaces (tenants).
-This dashboard complies with Sensu RBAC, so individual users will see summary information relevant to their access.
+## The `sensuctl` Command Line Interface (CLI)
 
-![The Sensu dashboard](img/03-01-dashboard.png)
+The `sensuctl` tool, short for _Sensu Control_, gives you full control of your Sensu pipeline from a command-line environment. 
+You can use `sensuctl` interactively in a shell, or script it as part of an automated solution.
 
-_**Figure 1:** the Sensu dashboard._
+The `sensuctl` tool is available for Linux, macOS, and Windows.
 
-### Namespaces switcher
+### EXERCISE 1: Install the `sensuctl` CLI
+#### Scenario
 
-Beyond the overview dashboard, most Sensu data is accessed on a per-namespace basis.
-To select or switch namespaces, press the "Select Namespace" option from the sidebar, or press the `Control + k` keyboard shortcut from anywhere in the Sensu web app.
+You want to view and manage resources in Sensu from a command-line environment or in an automated script.
 
-![The namespace switcher modal](img/03-02-namespace-switcher.png)
+#### Solution
 
-_**Figure 2:** the namespace switcher._
+Install the `sensuctl` CLI tool.
 
-![The "Select Namespace" option in the navigation sidebar](img/03-03-select-namespace.png)
+First, we will configure some helpful environment variables, download the `sensuctl` archive, then unpack that to a standard location on your system.
```

Comment:
This is grammatically correct, though it's a niche case with some nuance. Here `then` is being used as a _coordinating conjunction_, similar to `and`. In this usage, it follows the same rules as `and` which means it's only required once, prefixing the last phrase. In fact, `I picked up the pen, wrote a note, and then signed my name.` is grammatically equivalent to `I picked up the pen, wrote a note, then signed my name.`. We can omit the `and` in this scenario because it's implied by the presence of `then`. That's because when `then` is used as a coordinating conjunction it has a stricter meaning than `and`. There is an implied order to `then` conjunctions, but no such order is required for `and`. In order to be both more precise and more concise, it's better to use the shorten form where possible.